### ISSUE TITLE
There should be no requests list

### ISSUE THREAD

**Issue by jewpaltz** [anil-deepak/The-Exercise-App#4]
https://github.com/anil-deepak/The-Exercise-App/blob/8581711a64a95b3020635e5a95289c072a2bc8ff/server/models/users.js#L149

There is no point to the `requests` list.
Once you have that information in the `friends` list (with isApproved: false). You don't ever need the `requests` list.

**Comment by anil-deepak** [comment]
I wanted Requests is for requests that are received

**Comment by jewpaltz** [comment]
Yes, but that information is already stored in the friends list. So this list is duplicate. It does not give you anything extra.
